src/components/content/order/migrate/useMigrateServiceQuery.ts

```diff
@@ -75,10 +52,33 @@ export function useServiceDetailsPollingQuery(
         },
         enabled:
             serviceId !== undefined &&
-            (currentMigrationStatus === migrationStatus.MIGRATION_COMPLETED ||
-                currentMigrationStatus === migrationStatus.DEPLOY_FAILED ||
-                currentMigrationStatus === migrationStatus.DESTROY_FAILED),
+            (currentMigrationTaskStatus === taskStatus.SUCCESSFUL || currentMigrationTaskStatus === taskStatus.FAILED),
         staleTime: Infinity,
         gcTime: Infinity,
     });
 }
+
+export function useLatestServiceOrderStatusQuery(
```

Comment:
there is already one useLatestServiceOrderStatusQuery in useServiceDetailsPollingQuery.ts
We must reuse it. 
if possible you can move useLatestServiceOrderStatusQuery out of useServiceDetailsPollingQuery.ts to another file and use it from everywhere.

Reply:
good idea, let me udate this